<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="descriptionDisplay">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>320</y>
      <width>371</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="genVid">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>610</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Send to Video Generator</string>
    </property>
   </widget>
   <widget class="QLabel" name="title_final">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>411</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Title:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Thumbnail</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>300</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editTitle">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>187</y>
      <width>351</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>47</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Title</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>663</x>
      <y>0</y>
      <width>20</width>
      <height>691</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Tags</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateTags">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Tags</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tagkeyword">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tag Keyword</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLabel" name="videoQuestion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Video Script:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tagDisplay">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>180</y>
      <width>201</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="tagsLabel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tags for:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="descriptionEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>510</y>
      <width>511</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="totalwords">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Words:</string>
    </property>
   </widget>
   <widget class="QLabel" name="estvidtime">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Estimated Video Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="commentthreads">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Comment Threads:</string>
    </property>
   </widget>
   <widget class="QLabel" name="commentamount">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Comments:</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalaudiolines">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Total Audio Lines:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openeditor">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Go Back to Editor</string>
    </property>
   </widget>
   <widget class="QComboBox" name="thumbnailselect">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>211</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="usethumbnail">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>410</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Use Thumbnail</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Thumbnail Folder</string>
    </property>
   </widget>
   <widget class="QLabel" name="thumbnailarea">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>90</y>
      <width>371</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>Thumbnail</string>
    </property>
   </widget>
   <widget class="QLabel" name="thumbnailfolderpath">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>450</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Path To Latest Generated</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="openthumbnailfolder">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>191</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Thumbnail Folder</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportVideoFile">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>610</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Export Video File</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="tags">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Youtube allows 450 Characters</string>
    </property>
   </widget>
   <widget class="QLabel" name="charCount">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>280</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Characters: </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Youtube allows 100 Characters</string>
    </property>
   </widget>
   <widget class="QLabel" name="charCounttitle">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Current Characters:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1180</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
